archon.sh

```diff
@@ -12,6 +12,12 @@
 #
 #
 
+
+## ADDITION START
+sudoers_file=/etc/sudoers
```

Comment:
Tip: There is no need to save a value in a variable if it is going to be used once :)